File: week6/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    # Ask for size in [1:8]
    n = get_int("Size: ")
    while n < 1 or n > 8:
        n = get_int("Size: ")

    # Make a row
    for i in range(n):
        # Make a column
        # Make padding spaces
        for j in range(n - i - 1):
            print(" ", end="")

        # Make blocks
        for j in range(i + 1):
            print("#", end="")

        # Central spacer
        print("  ", end="")

        # Make second side blocks
        for j in range(i + 1):
            print("#", end="")

        # Next line
        print("")


if __name__ == "__main__":
    main()

File: week1/ps1/mario/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt picking a number [1, 8]
    int n;
    do
    {
        n = get_int("Height:");
    }
    while (n < 1 || n > 8);

    // Make a row
    for (int i = 0; i < n; i++)
    {
        // Make a column

        // Make padding spaces
        for (int j = 0; j < (n - i - 1); j++)
        {
            printf(" ");
        }
        // Make blocks
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("  ");
        // Make second side blocks
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // Next line
        printf("\n");
    }
}